### PR TITLE
feedback

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: src/styles/style.css [hunk 1]
```diff
@@ -239,15 +239,16 @@ img {
 
 .cssdb-feature-specification {
 	display: flex;
+	flex-wrap: wrap;
```
Comment (romainmenke) [inline src/styles/style.css:242]: This didn't flow right on mobile.

File: src/styles/style.css [hunk 2]
```diff
@@ -239,15 +239,16 @@ img {
...
 	font-size: 87.5%;
 	font-weight: 700;
 	margin-block: 0 -10.5px;
 	padding-inline: 15px;
 	position: relative;
 	text-transform: uppercase;
 
-	& > :not(:first-child) {
-		margin-inline-start: .25em;
+	& > :not(:last-child) {
```
Comment (romainmenke) [inline src/styles/style.css:250]: Better multi line placement.

File: src/styles/style.css [hunk 3]
```diff
@@ -428,8 +429,8 @@ img {
 		display: none;
 	}
 
-	& + .cssdb-browser {
-		margin-inline-start: 6px;
+	& + .cssdb-browser:not(:last-child) {
```
Comment (romainmenke) [inline src/styles/style.css:432]: Better multi line placement.

**Comment by Antonio-Laguna** [comment]
❤️  love this! Thanks

**Pull request merged**